src/sinks/util/mod.rs

```diff
@@ -164,9 +164,3 @@ impl<T> ElementCount for Vec<T> {
         self.len()
     }
 }
-
-impl ElementCount for serde_json::Value {
-    fn element_count(&self) -> usize {
-        1
-    }
-}
```

Comment:
This seems unrelated, but since it doesn't cause any problems… 👍🏻

Reply:
Yeah, I tried to be as radical as possible in a4ecba6b56e8e74eea73b2db22ea2cf4aca0720f to investigate how this is used. Can always add it back when we need it, otherwise I lean to say this is more of a distraction.